### PR TITLE
WIP: discussion of changes made for build from source DO NOT MERGE

### PR DESCRIPTION
**Pull request by mgianatagh**

### CODE REVIEW

File: Dockerfile_verrazzano [hunk 1]
```diff
@@ -0,0 +1,40 @@
+FROM container-registry.oracle.com/os/oraclelinux:7.8 as build_base
```
Comment (markxnelson) [inline Dockerfile_verrazzano:4]: need copyright/license headers - and i now realize i might have neglected that on the earlier ones too :(

File: verrazzano/definer.go [hunk 2]
```diff
@@ -0,0 +1,104 @@
+// We need the word "Copyright" here so this file will pass the license check
```
Comment (markxnelson) [inline verrazzano/definer.go]: put real notices in please

Comment (markxnelson) [reply verrazzano/definer.go]: did we write this? or copy it from somewhere?  if we copied it, we need to include attributions

Comment (markxnelson) [reply verrazzano/definer.go]: it looks copied to me

File: verrazzano/definer_test.go [hunk 3]
```diff
@@ -0,0 +1,82 @@
+// We need the word "Copyright" here so this file will pass the license check
```
Comment (markxnelson) [inline verrazzano/definer_test.go]: fix

**Review by markxnelson — CHANGES_REQUESTED** [review]

File: verrazzano/definer.go [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package verrazzano
+
+import (
+	"bytes"
+	"io"
+)
+
+// This class is typically wrapped around a TypeFilteringReadCloser, which
```
Comment (markxnelson) [inline verrazzano/definer.go:11]: did we write this? it looks suspicious.  if we copied it from somewhere, it needs an attribution

**Review by markxnelson — COMMENTED** [review]
deja vu

File: verrazzano/definer.go (hunk 2 above)
Comment (mgianatagh) [reply verrazzano/definer.go]: I confirmed with Jeff Berkowitz that he wrote these modules.

File: verrazzano/definer.go (hunk 4 above)
Comment (mgianatagh) [reply verrazzano/definer.go:11]: yes, I confirmed that we wrote this

**Pull request closed without merging**